### PR TITLE
ociswap volume and fee dimensions

### PR DESCRIPTION
**Pull request by ChrisPantostin**

### CODE REVIEW

**Comment by llamatester** [comment]
The ociswap adapter exports: 
        
 
 _______________________________________
    
    
    
    ERROR: Use Adapter v2 format for new adapters

File: dexs/ociswap/index.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        return {
+            dailyVolume: `${dailyVolume}`,
+            totalVolume: `${totalVolume}`,
+            timestamp
+        };
+    };
+};
+
+const adapters: BreakdownAdapter = {
+    breakdown: {
```
Comment (dtmkeng) [inline dexs/ociswap/index.ts:37]: add: `version: 2`

File: fees/ociswap.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        return {
+            dailyFees: `${dailyFees}`,
+            totalFees: `${totalFees}`,
+            timestamp
+        };
+    };
+};
+
+const adapters: BreakdownAdapter = {
+    breakdown: {
```
Comment (dtmkeng) [inline fees/ociswap.ts:37]: add: `version: 2`

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by ChrisPantostin** [comment]
Was closed in favor for this pr: 

https://github.com/DefiLlama/dimension-adapters/pull/1489

**Pull request closed without merging**